src/SqlStreamStore.Postgres/PostgresStreamStore.cs

```diff
@@ -78,14 +78,14 @@ public async Task CreateSchemaIfNotExists(CancellationToken cancellationToken =
             using(var connection = _createConnection())
             {
                 await connection.OpenAsync(cancellationToken).ConfigureAwait(false);
-                using(var transaction = connection.BeginTransaction())
+                using(var transaction = await connection.BeginTransactionAsync(cancellationToken).ConfigureAwait(false))
                 {
                     using(var command = BuildCommand($"CREATE SCHEMA IF NOT EXISTS {_settings.Schema}", transaction))
                     {
                         await command.ExecuteNonQueryAsync(cancellationToken).ConfigureAwait(false);
                     }
 
-                    using(var command = BuildCommand(_schema.Definition, transaction))
+                    using(var command = BuildCommand(_schema.Definition(_settings.Version), transaction))
```

Comment:
Crashes since return type of functions is edited:
![image](https://user-images.githubusercontent.com/6287467/200804368-01f074dd-8f68-4feb-86d1-580e0dd53aca.png)

Reply:
Migration created